config-files/usr/share/plasma/look-and-feel/org.openSUSE.desktop/contents/splash/Splash.qml

```diff
@@ -82,7 +115,7 @@ Rectangle {
         Image {
             id: busyIndicator
             anchors.horizontalCenter: parent.horizontalCenter
-            y: parent.height * 0.7
+            y: parent.height * 0.75
             source: "images/busywidget.svgz"
```

Comment:
I think this file was forgotten. Shouldn't it be added in the commit, too?

Reply:
It's per design, it should use files from upstream theme and replace just provided ones

Reply:
ah ok, I moved away the directory when testing your changes and I should have just copied it over